src/util/http.ts

```diff
@@ -58,3 +59,11 @@ export const createErrorRes = ({
     }),
   };
 };
+
+export const decodeToken = ((token : any) => {
```

Comment:
```suggestion
export const decodeToken = ((token : string) => {
```
토큰 값의 타입을 지정해주는 것이 추후, 함수를 가져와서 사용할 때 정적타이핑도구로써의 효용을 발휘하기 좋습니다.

Reply:
개발후 타입 어떻게 오는지 보고 바꿀려고 했는데 까먹고 있었네요 수정해놓겠습니다